Makefile

```diff
@@ -161,14 +161,14 @@ openrpc-gen:
 lint-imports:
 	@echo "--> Running imports linter"
 	@for file in `find . -type f -name '*.go'`; \
-		do goimports-reviser -list-diff -set-exit-status -company-prefixes "github.com/celestiaorg"  -project-name "github.com/celestiaorg/"$(PROJECTNAME)"" -output stdout $$file \
+		do goimports-reviser -list-diff -set-exit-status -company-prefixes "github.com/celestiaorg" -project-name "github.com/celestiaorg/celestia-node" -output stdout $$file \
```

Comment:
You don't need to change that

Reply:
If you store code in folder named differently (me) it won't work with value taken from `pwd`

Reply:
I happen not to and didn't change it, because there were no reason to do it. Anyway imports-reviser prefix is static and should be attached to github repo path, instead of local folder path.